airbyte-integrations/connectors/source-sentry/source_sentry/manifest.yaml

```diff
@@ -1,105 +1,149 @@
-version: "0.29.0"
+version: 0.57.0
```

Comment:
I'm a bit confused by this, and I see this in a couple PRs (another one is #35776). Should `manifest.version` be the minimal version of the CDK that can run this, or should it be the version of the connector? If it's the connector version, should this be 1.0.0?

Reply:
This version was added by the connector builder, this is version of cdk, I'm confused with this too, because poetry lock declares another version of cdk. So it looks like I need to update version in manifest.